doc/classes/DampedSpringJoint2D.xml

```diff
@@ -78,12 +78,16 @@
 	</methods>
 	<members>
 		<member name="damping" type="float" setter="set_damping" getter="get_damping">
+			The damping ratio of the spring joint. A value of 0 indicates an undamped spring, while 1 causes the system to reach equilibrium as fast as possible (critical damping).
```

Comment:
Per Writing Guidelines, avoid the "of the" construction - it just leads to longer sentences. Instead use "The spring joint's damping ratio."

You need to indicate what "undamped spring" means.

Reply:
oh you're right, thanks :)